Assignment2B/JobScheduler.java

```diff
@@ -0,0 +1,269 @@
+package Assignment2B;
+
+import java.io.IOException;
+
+/**
+ * 
+ * JobScheduler class used for First Come First Serve job scheduling it contain
+ * methods to calculate waiting time,completion time, turn around time,average
+ * waiting time, etc
+ *
+ */
+public class JobScheduler {
+
+	/**
+	 * @param process take two dimension array which contain arrival time and burst time of n process
+	 * @param numberOfProcess take number of process in Integer
+	 * @return an one dimension array which contain waiting time for all process
+	 */
+	public int[] getWaitingTime(int process[][], int numberOfProcess) throws IOException {
+		
+		//input validation
+		boolean validInput = inputValidation(process,numberOfProcess);
+		if(validInput == false){

```

Comment:
Use boolean input directly in condition like if(validInput) or if(!validInput) .

Reply:
i will from now on